scripts/overlap/kubectl-kubestellar-get-config-for-space

```diff
@@ -78,25 +78,25 @@ if [ -z "$sm_core_config" ]; then
     exit 1
 fi
 
+if [ -z "$space_config_file_name" ]; then
```

Comment:
Since this is a required parameter I would make it positional rather than a flag.